WIP: Basic structure for showcasing community contributions

cwognum opened the pull request:
Checklist:

- [X] Was this PR discussed in a issue? It is recommended to first discuss a new feature and let the community know whether you are planning or have started working on it before opening a PR.
- [ ] Add tests to cover the fixed bug(s) or the new introduced feature(s) (if appropriate).
- [X] Update the API documentation if a new function is added or an existing one is deleted.
- [x] Added a `news` entry.
  - _copy `news/TEMPLATE.rst` to `news/my-feature-or-branch.rst`) and edit it._

---

ping @maclandrol. I proposed a basic structure so we can discuss! What do you think?

cwognum commented:
For now, I was thinking of having a table with a row per contribution. This will not scale, but is a sensible solution for the short-term.

maclandrol commented on docs/community/get_involved.md:
```diff
@@ -0,0 +1,21 @@
+## Contribute
+The most straight-forward way to contribute is through Github. In short, you can help by: 
+
+- :star: **Star** the repository on Github
```
Add answering questions

maclandrol commented on docs/community/get_involved.md:
```diff
@@ -0,0 +1,21 @@
+## Slack
+[![slack](https://img.shields.io/badge/Join%20our%20Slack-4A154B?style=for-the-badge&logo=slack&logoColor=white)](https://join.slack.com/t/m2d2group/shared_invite/zt-16w1rjqqs-n81TiK~iB23XbZ0QWMYs~A)
+
+`molfeat` is an initiative that originated in the [M2D2 community](m2d2.io). To orchestrate these efforts the `#molfeat` channel 
```
Instead, the molfeat community is part of the m2d2 community

maclandrol commented on docs/developers/contribute.md:
```diff
@@ -1,9 +1,17 @@
 
+!!! note annotate "Get inspired by community contributions"
+    As we love community contributions, we decided to create some dedicate space in our documentation to highlight and celebrate such contributions.
+    Visit [Community Contribitons](../community/contributions.md) to learn more!s
```
There is a typo at the end.

maclandrol commented on docs/developers/contribute.md:
```diff
@@ -1,9 +1,17 @@
-Welcome to the molfeat community! We appreciate everyone's contribution and welcome all. Apart from code contributions, there are various other ways to support the community such as answering questions, providing assistance to others, enhancing the documentation, giving a ⭐️ or sharing the project.
+Welcome to the molfeat community! We appreciate everyone's contribution and welcome all. Apart from code contributions, there are various other ways to support the community. In short, you can: 
+
+- :star: **Star** the repository on Github
```
This seems like a duplicate of the *ways to contribute section* below. Would likely make sense to pull that section up here and/or rephrase both a bit

maclandrol commented on README.md:
```diff
@@ -110,9 +110,11 @@ model_card.usage()
 Please cite Molfeat if you use it in your research: [![DOI](https://zenodo.org/badge/613548667.svg)](https://zenodo.org/badge/latestdoi/613548667).
 
-## Contribute
+## Contribute and Community
```
Create subsections  maybe ?  Or just contribute is enough

maclandrol commented on README.md:
```diff
@@ -110,9 +110,11 @@ model_card.usage()
-See [developers](docs/developers/) for a comprehensive guide on how to contribute to `Molfeat`
+See [developers](docs/developers/) for a comprehensive guide on how to contribute to `molfeat`. `molfeat` is a community-led
+initiative and whether you're a first-time contributor or an open-source veteran, this project greatly benefits from your contributions.
+To learn more about the community that is behind molfeat, please see [community](docs/community/).
```
Community and ecosystem of tools ? Maybe in community we can say things like if you like molfeat, you will like datamol and xyz ?

maclandrol commented on docs/community/contributions.md:
```diff
@@ -0,0 +1,11 @@
+  <img src="https://contrib.rocks/image?repo=datamol-io/molfeat" />
+</a>
+
+Made with [contrib.rocks](https://contrib.rocks).
```
You can remove the contrib rocks part, it's fine.

maclandrol commented on docs/community/contributions.md:
```diff
@@ -0,0 +1,11 @@
+# Community contributions
+## Plugins
+
+## Tutorials
```
I like the split

maclandrol commented:
> For now, I was thinking of having a table with a row per contribution that links to an external page (e.g. Github of a plugin, blog post, etc.). Or would you prefer copying as much as possible over to molfeat instead of having a link?

No, the table is perfect. We can also add some links or some badges in the original tutorials too.

maclandrol commented:
Also ping @zhu0619

cwognum commented:
@maclandrol I addressed your comments! The community contributions page probably still needs some context descriptions, but I'll do those later. In the meantime, could you add the IPU badges? I cannot see the raw README and thus cannot copy the IPU badge (I saw you edited the description, so you probably can?).

cwognum commented:
**Reference in existing documentation:**

![image](https://github.com/datamol-io/molfeat/assets/28000055/901cbe66-acc5-4a71-884c-ca5a6d8d7fcf)

**New community contributions table**

![image](https://github.com/datamol-io/molfeat/assets/28000055/4d947cdc-cd9e-4d19-85c2-f069f4ece84e)

maclandrol reviewed (approved):
LGTM, thanks @cwognum. 
Can be merged !
